From c82b2b39e075a83ae4bd68a64054a62feb141b59 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:11:41 -0400
Subject: [powerpc] Bump BOOT_COMMAND_LINE_SIZE to 2048

Message-id: <1410545655-205645-473-git-send-email-dzickus@redhat.com>
Patchwork-id: 94462
O-Subject: [RHEL7 PATCH 472/626] powerpc: Bump BOOT_COMMAND_LINE_SIZE to 2048
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 579a53cafdd641f2d67ba4f4fbd9cc4180b8717d
Author: Anton Blanchard <anton@samba.org>
Date:   Mon Apr 14 21:55:25 2014 +1000

    powerpc: Bump BOOT_COMMAND_LINE_SIZE to 2048

    Bump the boot wrapper BOOT_COMMAND_LINE_SIZE to match the
    kernel.

    Signed-off-by: Anton Blanchard <anton@samba.org>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/boot/ops.h b/arch/powerpc/boot/ops.h
index 339e041..8aad3c5 100644
--- a/arch/powerpc/boot/ops.h
+++ b/arch/powerpc/boot/ops.h
@@ -15,7 +15,7 @@
 #include "types.h"
 #include "string.h"
 
-#define BOOT_COMMAND_LINE_SIZE 512
+#define BOOT_COMMAND_LINE_SIZE 2048
 #define MAX_PATH_LEN  256
 #define MAX_PROP_LEN  256 /* What should this be? */
 
-- 
1.7.1